example/ck_tile/01_fmha/fmha_fwd.cpp

```diff
@@ -149,9 +171,17 @@ bool run(const ck_tile::ArgParser& arg_parser)
     if(scale == .0f)
         scale = 1.0 / ck_tile::sqrt(static_cast<float>(hdim_q)); // TODO: q ? v ?
 
-    float descale_q = arg_parser.get_float("descale_q");
-    float descale_k = arg_parser.get_float("descale_k");
-    float descale_v = arg_parser.get_float("descale_v");
+    bool squant = arg_parser.get_bool("squant");
+    if constexpr(!std::is_same_v<DataType, ck_tile::fp8_t>)
+        assert(!squant);
```

Comment:
report "unsupported squant=1 for this data type" via stderr, instead of `assert(false)` here

Reply:
213826f32f284fad0e9bb548cb41939025b39132